equinix-openapi-fabric/docs/Account.md

```diff
@@ -0,0 +1,17 @@
+
```

Comment:
It would be helpful to include in this PR, for inclusion in the release notes, what models were added, renamed, removed, and where the model is an existing one what properties and public methods were added, renamed, and removed. These could link to the markdown doc, which is something we can do later in the release notes since the PR based links could break.

Largely inspired by https://developer.hashicorp.com/terraform/plugin/best-practices/versioning#changelog-specification, this could look like:

##  Model Changes
- `Account` model added
- `AdvanceConfiguration` model added
- `ConnectionLink` model added
- `CloudRouterAPI` `*Gateway*` methods renamed to `*CloudRouter`
- ... renamed to ...
- ... removed
- `Foo` fields `...`, `...` added / removed / renamed
- `FooAPI` `fooByFoo` method added 

If there is a Java package differ tool available, we could ideally automate this.